templates/form-element.html.twig

```diff
@@ -0,0 +1,59 @@
+{#
+/**
+ * @file
+ * Theme override for a form element.
+ *
+ * @todo Remove when https://www.drupal.org/node/3010558 is fixed.
+ *
+ * @see template_preprocess_form_element()
+ */
+#}
+{%
+  set classes = [
+    'js-form-item',
+    'form-item',
+    'js-form-type--' ~ type|clean_class,
```

Comment:
Let's not rename the js- prefixed classes because that would require us to rewrite selectors in JS

Reply:
Yepp. Some core js breaks as well :(